docs/source/en/tutorials/autopipeline.md

```diff
@@ -12,112 +12,52 @@ specific language governing permissions and limitations under the License.
 
 # AutoPipeline
 
-Diffusers provides many pipelines for basic tasks like generating images, videos, audio, and inpainting. On top of these, there are specialized pipelines for adapters and features like upscaling, super-resolution, and more. Different pipeline classes can even use the same checkpoint because they share the same pretrained model! With so many different pipelines, it can be overwhelming to know which pipeline class to use.
+[AutoPipeline](../api/models/auto_model) is a *task-first* pipeline that automatically selects the correct pipeline subclass based on the task. It handles the complexity of loading different pipeline subclasses without needing to know the specific pipeline subclass name.
```

Comment:
oh I think AutoPipelline select based on task & model; DiffusionPipeline only based on model. So AutoPipeline take just one step further, here it sounds like they are two parallel approach

so basically for DiffusionPipeline

```
model -> pipeline subclass 
```


for Auto
```
model -> pipeline subclass ->pipeline subclass within the same pipeline group for a different task

```